image_client/PIC_undo_batch.py

```diff
@@ -81,7 +84,32 @@ def sql_tree_purger(self, database, table, timestamp1, timestamp2):
 
         cursor.close()
 
-    def casbotany_csv_purger(self, database: str, table: str, MD5: str):
+    def batch_log_clear(self, table, MD5: str):
```

Comment:
function added to clear picbatch database with new connection, after main db is cleared, the file then removes the upload record from the picbatch db, to avoid clutter from old upload MD5 codes.